server/models/recipeModel.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const recipeSchema = new Schema({
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
```

Comment:
This should probably be associated with a Chef profile, not a user?

Reply:
Hey @jessdesr , Thanks for your time.
 That makes sense. I saw in the sketch that when we are fetching all recipes, we will want the name of the chef as well, that's why I connected them this way. But now I see we will need location and picture as well, so it makes sense to have it linked with chefProfile. 

Also I think I need to have the name field in the profile too. for rendering the recipes in "browse-meal" we will need that info and getting that separately from user model will be complicated. Actually I cannot see how that will be accomplished.

Reply:
That's a good point.... I think for now we can merge this in, then think about solution for browsing meals

Reply:
Sure. So I will modify the recipe user field to be linked to chefProfile and I will merge.